test/unit/controllers/InventoryLinkingExportControllerSpec.scala

```diff
@@ -45,7 +45,7 @@ class InventoryLinkingExportControllerSpec extends UnitSpec
   with Matchers with MockitoSugar with BeforeAndAfterEach {
 
   trait SetUp extends AuthConnectorStubbing {
-    override val mockAuthConnector: InventoryLinkingAuthConnector = mock[InventoryLinkingAuthConnector]
+    override val mockAuthConnector: AuthConnector = mock[AuthConnector]
 
```

Comment:
Can you do me favor pls and fix some unrelated code. `mockExportsLogger2` should be `mockExportsLogger`